### PR TITLE
Finding equilibrium point in the array in Python

### PR DESCRIPTION
**Pull request by sonali-sk**
Have you read the [Contributing Guidelines on Pull Requests]

YES

Description

Given an array of n positive integers, find the first equilibrium point in the array.
Equilibrium Point in an array is a position at which sum of elements before it is equal 
to the sum of elements after it.

Checklist

- [YES ] I've read the contribution guidelines.
- [YES ] I've checked the issue list before deciding what to submit.
- [ YES] I've edited the `README.md` and link to my code.

Related Issues or Pull Requests

Issue #7381

### CODE REVIEW

**Comment by sonali-sk** [comment]
@HarshCasper @ankitaggarwal23 @Apoorve73 @siddharth25pandey

File: Python/other/equilibrium_point_in_array.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Equilibrium point does not exist.
+
+Test case 2
+Input:
+Enter elements of the array: 3 2 4 1 4
+Output:
+Position 3 is the equilibrium point.
+
+"""
+
```
Comment (siddharth25pandey) [inline Python/other/equilibrium_point_in_array.py]: Add Time and Space Complexity

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@sonali-sk   Do the suggested changes

**Comment by sonali-sk** [comment]
@siddharth25pandey changes made. Please review.

File: Python/other/equilibrium_point_in_array.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[7]:
+
+
```
Comment (ankitaggarwal23) [inline Python/other/equilibrium_point_in_array.py]: Remove this redundant lines

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by sonali-sk** [comment]
@ankitaggarwal23 @siddharth25pandey changes made. Please review.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**